features/admin.branding.v1/components/branding-preference-tabs.tsx

```diff
@@ -21,6 +21,7 @@ import { AppState } from "@wso2is/admin.core.v1/store";
 import { commonConfig } from "@wso2is/admin.extensions.v1/configs";
 import FeatureFlagLabel from "@wso2is/admin.feature-gate.v1/components/feature-flag-label";
 import FeatureFlagConstants from "@wso2is/admin.feature-gate.v1/constants/feature-flag-constants";
+import "./branding-preference-tabs.scss";
```

Comment:
let's move the stylesheet imports to the bottom of the import statement list for better readability and consistency

Reply:
done!